src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+  color: $main-text-color;
```

Comment:
Ensure that the SCSS variable `$main-text-color` is defined in your SCSS files. If not, it will cause a compilation error.